README.md

```diff
@@ -78,25 +99,60 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// Read
+app.get("/candidates", function(req, res){
+  //.find all candidates then
+  //render to candidates index
+  //use handle bars so it can be used in view
+});
+app.get("/candidates/:name", function(req, res){
+
+});
 
+//Create
+app.post("/candidates", fucntion(req, res){
+  Candidate.create(req.body.candidate).then(function(candidate){
+    //redirect
+  })
+})
+
+//Update
+app.post("/candidates/:name", funciton(req, res){
+  Candidate.findOneAndUpdate(
+    //use body-parser and redirect.
+  )
+})
+
+//Delete/Detroy
+app.post("/candidates/:name/delete", function(req, res){
+  Candidate.findOneAndRemove(
+    //use body-parser and redirect
+  )
+})
```

Comment:
:+1: Although Express does have `.put` and `.delete`. We just don't use when it purely dealing with HTML forms since they only support `.post`.